test/validation_manifold.jl

```diff
@@ -118,7 +118,7 @@ end
         @test injectivity_radius(A, x, CustomArrayManifoldRetraction()) == 11
```

Comment:
Renaming `CustomArrayManifoldRetraction` to `CustomValidationManifoldRetraction` would make it a bit more consistent.